### ISSUE TITLE
Continuous Integration Failing Erroneously

### ISSUE THREAD

**Issue by marstr** [Azure/azure-sdk-for-go#485]
[Repro in PR](https://github.com/Azure/azure-sdk-for-go/pull/484)

When submitting even trivial pull-requests, continuous integration is failing. Digging in a little, it seems that the "gas" component started mistakenly detecting security vulnerabilities in example files that have hard-coded faux passwords. The cause of this regression is unknown, but I suspect that it was a bug introduced by an external dependency (like gas itself) as these issues started independently of any commit made in this repository.

**Comment by marstr** [comment]
I have mitigated this issue by temporarily disabling gas checks during continuous integration. IT IS NOT YET SOLVED. However, because it is no longer blocking PRs, I'll lower the priority to P1.

**Comment by marstr** [comment]
After digging into gas, I discovered that they changed the engine they use to resolve globs passed as arguments. This resulted in a slight change in behavior where, basically the regex used to evaluate paths changed from `/<somePattern>$/` to `/^<somePattern>$/`. I submitted an issue in the gas repository because of the breaking change, but also updated out .travis.yml file to accommodate this change in behavior. #487
